operator/v1/types.go

```diff
@@ -93,37 +93,37 @@ type KlusterletSpec struct {
 	// +required
 	WorkImagePullSpec string `json:"workImagePullSpec,omitempty" protobuf:"bytes,3,opt,name=workImagePullSpec"`
 
-	// ClusterName is the name of the spoke cluster to be created on hub.
-	// The spoke agent generates a random name if it is not set, or discovers the appropriate cluster name on openshift.
+	// ClusterName is the name of the managed cluster to be created on hub.
+	// The klusterlet agent generates a random name if it is not set, or discovers the appropriate cluster name on openshift.
```

Comment:
We should decide now whether we want to conventionally capitalize `klusterlet` or not.

Reply:
Good call. I opted for `Klusterlet`.